lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -287,23 +287,32 @@ export default function LaunchErrorDialog({
           title="Group set not found"
         >
           <p>
-            Hypothesis couldn&apos;t find this assignment&apos;s group set. This
+            This Hypothesis assignment was set up to use Canvas{"'"} Group Sets,
+            and we can no longer find the Group Set for this assignment. This
             could be because:
           </p>
 
           <ul className="px-4 list-disc">
             <li>The group set has been deleted from Canvas.</li>
-            <li>
-              This course was created by copying another course (Canvas
-              doesn&apos;t copy the group sets over when you copy a course).
-            </li>
+            <li>This course was created by copying another course.</li>
           </ul>
+          <p>
+            If the group set has been <b>deleted</b> from this course, an
+            instructor needs to edit the assignment settings and select a group
+            set.
+          </p>
 
           <p>
-            <b>
-              To fix this problem, an instructor needs to edit the assignment
-              settings and select a new group set.
-            </b>
+            If this is a <p>copied course</p> the instructor needs to create a
+            new group set in this course that matches the name of the group set
+            in the old course.
+            {error?.details?.group_set_name && (
+              <>
+                {' '}
+                The group set name in the old course was{' '}
+                {error.details.group_set_name}
```

Comment:
```
lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx(309,30): 
error TS2339: Property 'group_set_name' 
does not exist on type 'string | object'.
```


I reckon this type anno for this is: https://github.com/hypothesis/lms/blob/main/lms/static/scripts/frontend_apps/errors.ts#L43

Should I somehow assert this is the object part and ignore the other? How?

Reply:
@acelaya This is not ready for review but adding you here to take a look at this